src/NiceLinAlg.jl

```diff
@@ -1,10 +1,17 @@
 using LinearAlgebra: Hermitian, Symmetric, UpperTriangular, LowerTriangular
-using LinearAlgebra: _chol!, Cholesky, checkpositivedefinite
+using LinearAlgebra: _chol!, Cholesky, NoPivot, checkpositivedefinite
```

Comment:
this `using` should probably also be behind a static if.

Reply:
Of course!